src/tests/integration/test_generic_db_operations.py

```diff
@@ -613,7 +637,21 @@ def test_sqlcmd_tables_columns(
     assert out.result
 
 
-@pytest.mark.parametrize("ip_with_dynamic_db", ALL_DATABASES)
+@pytest.mark.parametrize(
+    "ip_with_dynamic_db",
+    [
+        ("ip_with_postgreSQL"),
+        ("ip_with_mySQL"),
+        ("ip_with_mariaDB"),
+        ("ip_with_SQLite"),
+        pytest.param(
+            "ip_with_MSSQL",
+            marks=pytest.mark.xfail(reason="Bug #428"),
+        ),
+        ("ip_with_MSSQL"),
```

Comment:
Why are we skipping ip_with_MSSQL and then running it?

Reply:
A typo, it would be `ip_with_duckdb`